lua/nvim-treesitter/parsers.lua

```diff
@@ -1564,6 +1564,24 @@ list.prisma = {
   maintainers = { "@elianiva" },
 }
 
+list.problog = {
+  install_info = {
+    url = "https://codeberg.org/foxy/tree-sitter-problog",
+    files = { "src/parser.c" },
+    branch = "feat/problog",
```

Comment:
No need to specify the branch if it's the default.

Reply:
You are right, thank you! I got confused with the local install (from init.lua)!